src/api-reference/request/v4.endpoints.workflow-resources.markdown

```diff
@@ -0,0 +1,174 @@
+---
+title: Travel Request v4 - Workflow Resources
+layout: reference
+---
+
+{% include prerelease.html %}
+
+* [Workflow Resources](#workflow-resource)
+  * [Move an existing Request in the approval workflow](#Move-existing-request-in-approval-workflow)
+
+## HTTP Status Codes
+
+To learn more about response HTTP status codes for this API see [Travel Request v4 - HTTP Status Codes](./v4.response-codes.html).
+
+# <a name="workflow-resource"></a>Workflow Resources
+
+Manage workflow transitions for a Request.
+
+## <a name="Move-existing-request-in-approval-workflow"></a>Move an existing Request in the approval workflow
+
+### Scopes
+
+`travelrequest.write` - Refer to [Scope Usage](./v4.get-started.html#scope-usage) for full details.
+
+### HTTP Request
+
+#### Parameters
+
+Name|Parameter Type|Data Type|Description
+---|---|---|---
+requestUuid|path|string|**Required** The unique identifier of the Request
+action|path|string|**Required** The state transition to be executed (`submit`, `approve`, `recall`, `sendback`, `cancel`, `close` or `reopen`)
+comment|path|string|Only works with when the workflow action is ‘sendback’. This comment is visible wherever Request comments are available to the employee, approver, and/or Request administrator
+userId|path|string|The unique identifier of the user performing the status transition. <br> **Required** when connecting with a Company token, if empty a 400 `missingRequiredParam` error code will be displayed
+companyID|path|string|The unique identifier of the company
+
+The HATEOAS links for actions available given the current user and state are listed in the operations of the Request resource.
+
+* Traveler actions
+  * submit: initiate the approval workflow
+  * recall: get back the Request, usually to modify the content
+  * cancel: cancel the Request and attached itineraries
+  * close: archive the Request
+  * reopen: get back an archived Request
+* Approver / Processor actions
+  * approve: move the Request to the next approval step
+  * sendback: reject the Request and send back to the Traveler
+
+##### URI Template
```

Comment:
This goes directly beneath the HTTP Request header.

Reply:
OK -> @laurentlhermine  can you please put the following part just under the HTTP Request subtitle? (instead of after the parameters as it is now)

"The HATEOAS links for actions available given the current user and state are listed in the operations of the Request resource.

* Traveler actions
  * submit: initiate the approval workflow
  * recall: get back the Request, usually to modify the content
  * cancel: cancel the Request and attached itineraries
  * close: archive the Request
  * reopen: get back an archived Request
* Approver / Processor actions
  * approve: move the Request to the next approval step
  * sendback: reject the Request and send back to the Traveler"

Reply:
@laurentlhermine : the v4_Request_workflow resources_v8.md file has been updated accordingly in https://jira.concur.com/browse/TVR-10569

Reply:
done